test/Hyperwallet.spec.js

```diff
@@ -2303,7 +2355,9 @@ describe("Hyperwallet", () => {
         /** @test {Hyperwallet#listPrepaidCards} */
         it("should do get call with options", () => {
             const callback = () => null;
-            const options = { status: "test-status", createdBefore: "test-createdBefore", createdAfter: "test-createdBefore", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit" };
+            const options = {
+                status: "test-status", createdBefore: "test-createdBefore", createdAfter: "test-createdBefore", sortBy: "test-sortBy", offset: "test-offset", limit: "test-limit",
```

Comment:
shouldn't it be formatted in many lines?

Reply:
Hmm, I think it only moves it if they're not all on the same line
```
ESLint: Object properties must go on a new line if they aren't all on the same line.(object-property-newline)
```